helpers/compose.go

```diff
@@ -107,14 +139,95 @@ func Ps(ctx context.Context, c *client.Client, all, quiet bool, formatOpt string
 		}
 		return nil
 	}
+	return format.Print(containerList, formatOpt, os.Stdout,
```

Comment:
> Instead of passing os.Stdout here, you can allow Ps to take any io.Writer.
Can surely accept any `io.Writer` but this particular method here is to mimic `docker compose ps`([ref](https://github.com/docker/compose-cli/blob/main/cli/cmd/ps.go)). For using any other writer, `ListContainers` can be used to get the containers.

> Btw which is the use case of this function? Mesheryctl?
Yes